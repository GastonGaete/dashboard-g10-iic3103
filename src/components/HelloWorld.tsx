import { ReactElement } from 'react';
import { useTranslation } from 'react-i18next';

function HelloWorld(): ReactElement {
  const { t } = useTranslation();

  return (
    <div>
      <p>{t('HelloWorld.content')}</p>
    </div>
  );
}

export default HelloWorld;
