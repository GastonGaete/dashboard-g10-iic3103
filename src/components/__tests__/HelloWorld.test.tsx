import { render } from '@testing-library/react';
import HelloWorld from '../HelloWorld';

test('render <HelloWorld />', () => {
  const { container } = render(<HelloWorld />);
  expect(container).toMatchSnapshot();
});
