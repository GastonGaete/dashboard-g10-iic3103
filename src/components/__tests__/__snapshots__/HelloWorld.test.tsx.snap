// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <HelloWorld /> 1`] = `
<div>
  <div>
    <p>
      HelloWorld.content
    </p>
  </div>
</div>
`;
